about.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <script src="https://kit.fontawesome.com/d6cd3c7604.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <header class="first-head">
+        <ul id="first-head-list">
+          <li class="heads-list icon">
+            <a href="#"><i class="fa fa-facebook"></i></a>
+          </li>
+          <li class="heads-list icon">
+            <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+          </li>
+          <li class="heads-list"><a href="#">English</a></li>
+          <li class="heads-list"><a href="#">My Page</a></li>
+          <li class="heads-list"><a href="#">Logout</a></li>
+        </ul>
+      </header>
+
+
+      <div class="first-container">
+        <div class="first-image">
+          <div class="hamburger">
+            <span class="line"></span>
+            <span class="line"></span>
+            <span class="line"></span>
+          </div>
+          <nav class="second-nav">
+            <div class="nav-logo">
+              <a href="index.html"><img src="images/logo1.svg" alt="our logo" />
+              </a>
+            </div>
+  
+            <div class="nav-list">
+              <ul class="ul-items one">
+                <li class="list-item">
+                  <a class="link" href="index.html">About</a>
+                </li>
+                <li class="list-item">
+                  <a class="link" href="#program">Program</a>
+                </li>
+                <li class="list-item">
+                  <a class="link" href="#program">Join</a>
+                </li>
+                <li class="list-item">
+                  <a class="link" href="#program">Sponsors</a>
+                </li>
+                <li class="list-item">
+                  <a class="link" href="#program">Contact</a>
+                </li>
```

Comment:
- [ ] Kindly make sure when the user is on the about page, the link in the navigation bar is `home` to navigate on the home(`index`) page.
- [ ] Also, consider linking your link in the navigation bar with the corresponding section on the page. `i.e when the user clicks the link they are redirected to the exact section on the page(sponsors, program,...)`